wsd/COOLWSD.cpp

```diff
@@ -2997,6 +2997,23 @@ void COOLWSD::innerInitialize(Application& self)
     StorageBase::initialize();
 
 #if !MOBILEAPP
+    // Check for smaps_rollup bug there rewinding and rereading gives
+    // bogus doubled results
+    if (FILE* fp = fopen("/proc/self/smaps_rollup", "r"))
+    {
+        std::size_t nMemoryDirty1 = Util::getPssAndDirtyFromSMaps(fp).second;
+        std::size_t nMemoryDirty2 = Util::getPssAndDirtyFromSMaps(fp).second;
+        LOG_TRC("Comparing smaps_rollup read and rewind+read: " << nMemoryDirty1 << " vs " << nMemoryDirty2);
+        if (nMemoryDirty2 >= nMemoryDirty1 * 2)
```

Comment:
getPssAndDirty is unlikely to allocate too many pages (I guess) - and I expect it will work but I wonder how glibc and FILE * stuff buffering/allocation works etc. the >= *2 could potentially fail perhaps; would prefer a bit more slack myself for glibc to fool around allocating more space ;-) how about we do an extra defensive getPssAndDirtyFromSMaps() in between  nDirty1 and nDirty2 - and leave the >= 2 there ? =) would be more foolproof I think (?) =)

OH - and also, from a naming perspective - we don't hungarian mangle names in online be good to javaName them to stop the contagion spreading =)